### ISSUE TITLE
Add Categories

### ISSUE THREAD

**Issue by nickttng** [adunkman/confa11y#16]
**_Updated: June 14, 2017_**

More categories in addition to 'Live Captioning':

1. **Sign Language Interpreting**
2. **Mobility Access:** Reserved seating for those on mobility devices; wider pathways between chairs, tables, Restrooms include ADA stall w/ grab bars, lowered sinks, etc; if building itself is accessible (step-free access? if have steps, elevator or another way to other floors?) and whether this is mentioned on the conference/event website, FB event, etc.
3. **Blind/Vision Access:** Reserved seating close to stage; tactile + braille signs
4. **DeafBlind:** SSP ([Support Service Provider](http://www.aadb.org/information/ssp/ssp.html)) and pro-tactile sign language interpreting. For more info, @meltedspork is a good contact 
5. **Childcare / Child-Friendly Environment**
6. **Restrooms: All-Gender / Gender-Neutral**
7. **Nursing / Pumping Room**  (suggested by @iennae / March 30, 2017)
8. **Quiet / Rest Area:** Especially helpful for those dealing with chronic illness, ASD, taking medication in private, etc. Not for socializing 
9. **Service Animals Welcome** Also, does it have a park/area for relief and walking? (seen on DjangoCon site / May 1, 2017)
10. **Recovery Meeting**  (seen on DjangoCon site / May 1, 2017)
11. **Dietary Accommodation** (inspired by @stephmarx' talk, _Eating in Tech_; June 2017)

**Comment by iennae** [comment]
7. **Breast feeding/pumping room**

**Comment by nickttng** [comment]
Thanks for helping out. This is a great addition that I'll keep an eye out for and ask conferences if they offer this. If you know/find any, feel free to submit.

**Comment by adunkman** [comment]
Great recommendation @iennae! I know @asheren used to look for the same!

**Comment by iennae** [comment]
devopsdays Silicon Valley 2016 totally had a nursing/pumping room, Chef Conf 2017 is going to have one. WONTFIX 2017 had one (as it was held in the github offices which had one). As a nursing mom myself, it's pretty critical and makes it so much easier for me than having to ask about it.

**Comment by nickttng** [comment]
@iennae `Nursing / Pumping Room ` is now a category on the homepage. And just added these confs. Awesome.

**Comment by asheren** [comment]
rubyconf and railsconf every year have childcare and a lactation room, as does strangeloop.

**Comment by nickttng** [comment]
@asheren Just added these confs. For RailsConf site, love how childcare info is so easy to spot on the homepage.

**Comment by ismith** [comment]
Should 'Physical Access' be renamed 'Wheelchair Access' or maybe 'Mobility Access'?  It wasn't immediately obvious to me what 'Physical Access' meant.

Also, putting this list (& descriptions) in a post somewhere would be cool as a reference. Would also make it easier to have a mental checklist when adding a conf.

**Comment by nickttng** [comment]
Good point. 'Mobility Access' seems to be the better, broader label than 'Wheelchair Access' because it's not just limited to wheelchair, like knee walkers, rollators, etc. Thoughts?

Re: checklist, that's a great idea. I'll open up an issue to create a doc for people to find it easily.
